frontend/src/metabase/core/components/Slider/Slider.unit.spec.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import Slider from "./Slider";
+
+describe("Slider", () => {
+  it("should render 2 range inputs", () => {
+    const { container } = render(
```

Comment:
Usage of `container` is discouraged by the author of the react testing library. We are supposed to use `screen` from `@testing-library/react` instead: `screen.getByRole('textbox')`